src/index.ts

```diff
@@ -219,21 +219,22 @@ class ServerlessSSMPublish {
       {
         Name: param.path,
         Description: param.description || `Placed by ${this.serverless.service.getServiceName()} - serverless-ssm-plugin`,
-        Value: typeof param.value === 'string' ? param.value : yaml.safeDump(param.value),
+        Value: typeof param.value === 'string' ? param.type !== 'StringList' ? param.value : yaml.safeDump(param.value) : new Array(param.value).join(','),
```

Comment:
```suggestion
        Value: typeof param.value === 'string'
            ? param.value
            : param.type === 'StringList' && Array.isArray(param.value)
              ? param.value.join(',')
              : yaml.safeDump(param.value),
```

I think this is what you are going for here, but correct me if I'm wrong.
If the value is a string, we want to just pass that into the Value. 
If it isn't, previously we would use yaml.safeDump to preserve the yaml format. 
What we're adding now is that if the type given for the param is `StringList` and the value is given as an array in the yaml, i.e.
```
      - path: /service/test/arrayStringList
        type: 'StringList'
        value:
          - first
          - second 
          - third
```
we join it into a string to match AWS formatting